app/src/main/java/com/to_do_list/app/ListOnClickListener.kt

```diff
@@ -0,0 +1,5 @@
+package com.to_do_list.app
+
+interface ListOnClickListener {
+    fun onListClickListener(data: ToDoList)
+}
```

Comment:
Alt ser godt ud, dog check lige de ting jeg har ændret da der ser ud ud til at være conflicts. 

Jeg ville bare lige spørge hvorfor det er at der laves interface ListOnClickListener interface med in function med ToDoList parameter?

Reply:
Selvfølgelig, det er så vi kan få fat i den specifikke liste, som der bliver trykket på. Det gør det f.eks. muligt at parse den trykkede listes navn, når vi starter "ListOfToDos". Den bliver startet som en Itent i MainActivity Vi skal lige have parset ID'et på listen også, så vi kan hente alle TODOs i databasen, som er tilknyttet den liste.